tests/test_calculator.cc

```diff
@@ -24,6 +24,7 @@
  * Free Software Foundation, Inc., 59 Temple Place - Suite 330,
  * Boston, MA 02111-1307, USA.
  */
+#define BOOST_TEST_MODULE boost_test_sequence
```

Comment:
Could you remind me why this is needed? We don't use it in other files, right?

Reply:
I use boost::test_tools::per_element() to simplify test over collection collections so [boost needs this define to work](https://www.boost.org/doc/libs/1_65_0/libs/test/doc/html/boost_test/testing_tools/extended_comparison/collections.html)